src/System.Private.ServiceModel/tools/setupfiles/BridgeCertificateBootstrapper.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+show_usage() 
+{
+    echo ""
+    echo "WCF Core Bridge CA Certificate Bootstrapper" 
+    echo "    Usage: $0 [bridge-host] [certificate-file]"
+    echo ""
+    echo "    bridge-host - hostname where Bridge is running" 
+    echo "    certificate-file - Full file path where the retrieved certificate should be saved"
```

Comment:
NOTE: certificate-file probably will get hard-coded to ~/tmp/wcfbridge-ca.crt or some similar name and location. However, for debugging purposes, I allow myself the flexibility to stash the cert wherever I like so I can look at it.